test_signatures_similarity_cli.py

```diff
@@ -0,0 +1,38 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import download_data
+import crash_similarity
+import argparse
+import utils
+from datetime import timedelta
+
+parser = argparse.ArgumentParser(description = 'Test similarities between two signatures')
+parser.add_argument('-1','--one',required = True, help = 'First signature' )
+parser.add_argument('-2','--two',required = True, help = 'Second signature' )
+parser.add_argument('-p','--product',required = True, help = 'Product for which crash data is needed to be downloaded' )
+parser.add_argument('-t','--top', help = 'Number of top similar and different stack traces(Default 10)' )
+args = parser.parse_args()
+
+
+if __name__ == '__main__':
+    #download_data.download_crashes(days=1, product=args.product)
+    paths = download_data.get_paths(days=1, product=args.product)
+    #paths = ['crashsimilarity_data/firefox-crashes-2016-11-09.json', 'crashsimilarity_data/firefox-crashes-2016-11-08.json', 'crashsimilarity_data/firefox-crashes-2016-11-07.json', 'crashsimilarity_data/firefox-crashes-2016-11-06.json', 'crashsimilarity_data/firefox-crashes-2016-11-05.json', 'crashsimilarity_data/firefox-crashes-2016-11-04.json', 'crashsimilarity_data/firefox-crashes-2016-11-03.json']
+
+    corpus = crash_similarity.read_corpus(paths)
+
+    model = crash_similarity.train_model(corpus)
+
+    print(args.one + ' vs ' + args.two)
+    similarities = crash_similarity.signature_similarity(model, paths,'shutdownhang | js::GCMarker::processMarkStackTop','shutdownhang | js::GCMarker::processMarkStackTop')
+    #similarities = crash_similarity.signature_similarity(model, paths, args.one, args.two)
+    if(args.top == None):
```

Comment:
Nit: if args.top is None

The other option is to add a default value to `parser.add_argument('-t','--top'`: https://docs.python.org/2.7/library/argparse.html#default